huggingface/pytorch/buildspec-neuron.yml

```diff
@@ -2,8 +2,8 @@ account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
 region: &REGION <set-$REGION-in-environment>
 base_framework: &BASE_FRAMEWORK pytorch
 framework: &FRAMEWORK !join [ "huggingface_", *BASE_FRAMEWORK]
-version: &VERSION 1.9.1
-short_version: &SHORT_VERSION 1.9
+version: &VERSION 1.10.1
+short_version: &SHORT_VERSION 1.10
```

Comment:
Please change this to "1.10" and try it. The build should go through fine.

Reply:
changed it build still failed